### PR TITLE
translatable extension for collection, move methods from extension to helper

### PR DESCRIPTION
**Pull request by JarekW**

### CODE REVIEW

**Comment by chives** [comment]
@fsi-open

**Comment by rn0** [comment]
:+1:

**Comment by chives** [comment]
@fsi-open

**Comment by JarekW** [comment]
:+1:

File: features/fixtures/project/src/FSi/FixturesBundle/Entity/EventTranslation.php [hunk 1]
```diff
@@ -182,4 +199,24 @@ public function setEvent($event)
...
+     * @return ArrayCollection
+     */
+    public function getFiles()
+    {
+        return $this->files;
+    }
+
+    public function addFile(File $file)
+    {
+        $this->files->add($file);
```
Comment (szymach) [inline features/fixtures/project/src/FSi/FixturesBundle/Entity/EventTranslation.php]: shouldn't you be checking if the file is already in the collection?

**Comment by JarekW** [comment]
@fsi-open

**Comment by szymach** [comment]
:+1:

**Comment by rn0** [comment]
:+1:

**Pull request merged**